hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/token/HapiTokenUpdate.java

```diff
@@ -219,13 +226,33 @@ public HapiTokenUpdate expiry(long at) {
         return this;
     }
 
-    public HapiTokenUpdate improperlyEmptyingAdminKey() {
-        useImproperEmptyKey = true;
+    public HapiTokenUpdate properlyEmptyingAdminKey() {
+        useEmptyAdminKeyList = true;
         return this;
     }
 
-    public HapiTokenUpdate properlyEmptyingAdminKey() {
-        useEmptyAdminKeyList = true;
+    public HapiTokenUpdate usingInvalidAdminKey() {
```

Comment:
Handles invalid keys but what about "empty" keys?

Reply:
@thomas-swirlds-labs 
We separated the logic of the HIP into 2 PRs. This one contains the implementation of low prio keys to change themselves to another valid or an invalid key. The other functionality will be [here](https://github.com/hashgraph/hedera-services/pull/12271) where the "empty" keys are considered removed

Reply:
Ok. Makes sense. In that case it could also be useful to have a long running HIP-540 feature branch to merge the multiple PRs into before develop ?